windows_install.bat

```diff
@@ -0,0 +1,292 @@
+@echo off
+REM Pushes environment variables
+setlocal
+REM Allows us to properly set and use values of variables inside of if statements/for loops
+setlocal EnableDelayedExpansion
+
+REM If they have just cloned the repo, we want to ask them if they want to initialize
+REM the directory structure.
+REM If they already have the directory structure set up and they are running
+REM windows_install from the source folder, we want it to work without reinit'ing everything
+REM This can be implemented by checking if there are build*, install*, and source* 
+REM folders in the current directory. If so, you can continue building as normal.
+REM If not, then check if there are those folders in ..\.. in which case, cd there and
+REM start building as normal. If neither of those work, ask if they have just cloned
+REM roadrunner and they would like to initialize the directory structure.
+if exist "build*" if exist "install*" if exist "source*" goto :init_done
+if exist "..\..\build*" if exist "..\..\install*" if exist "..\..\source*" (
+  REM Move the current directory to the correct folder
+  echo Warning: This script is supposed to be executed from the top of the directory
+  echo          structure.
+  echo Warning: Changing directory to top of directory structure with `cd ..\..\`
+  cd ..\..\
+  goto :init_done
+)
+set INIT=
+echo It looks like you have just downloaded roadrunner and you are attempting to install.
+set /p INIT="Would you like to create the roadrunner build directory structure? Enter V to see a visualization of what will be created. [Y/N/V]
```

Comment:
for me the script terminates after this with

```
Would you like to create the roadrunner build directory structure? Enter V to see a visualization of what will be created. [Y/N/V]Y
The syntax of the command is incorrect.
```

the set /p actually succeeded and the variable is set. However the if statement afterwards fails. (i don't think there is an else if either.). I would just use conditional jumps with 2 labels, one printing the info, the other doing the work.

Reply:
Ah, the syntax error was when I added the visualization later. I have escaped the special characters and it now works on my machine